### ISSUE TITLE
Flutter build windows app not run when move it to different directory

### ISSUE THREAD

**Issue by Kaioken3000** [flet-dev/serious-python#93]
My flutter windows build application run normally after i build it, but when i copy the build directory to another folder it doesn't run.

i think it because something is wrong with the serius_python plugin, because it happend only after i add it to my code.

here the code:

Future<void> main() async {
  WidgetsFlutterBinding.ensureInitialized();
  speechToText();

  // runApp(const LoginPageAdmin());
  runApp(
    MultiProvider(
      providers: [
        ChangeNotifierProvider(create: (_) => Progress()),
      ],
      child: const LoginPageAdmin(),
    ),
  );
}

Future<void> speechToText() async {
  await SeriousPython.runProgram(
    "D:/D/demofastapi/main.pyc",
    sync: false,
  );
}

Please help!!!!

I just want it to run

**Comment by FeodorFitsner** [comment]
Will take a look.

Please do not create duplicates.

**Comment by FeodorFitsner** [comment]
What error do you get (in the console or a dialog)?

**Comment by ndonkoHenri** [comment]
In the meantime the Windows implementation has been rewritten twice. The C++ plugin that used to start Python is now just a registration shell; the interpreter lifecycle lives in `dart_bridge.dll` and is invoked from Dart over FFI (3.0.0), and since 4.0.0 the app and its packages ship unpacked next to the .exe. Every path the runtime uses is derived from `Platform.resolvedExecutable`, so copying the build output to another folder is supported by design.

Closing as obsolete against 0.7.x. If a relocated build still fails on 4.5.1+, please open a new issue with the console output or the text of the error dialog and we'll take a look.
